src/build.props

```diff
@@ -29,8 +29,8 @@
     place. These properties are actually used by the PowerShell script that
     hides the previous package versions on nuget.org.
     -->
-    <PreviousVersionPrefix>2.1.0</PreviousVersionPrefix>
-    <PreviousSchemaVersionAsPublishedToSchemaStoreOrg>2.1.0-beta.2</PreviousSchemaVersionAsPublishedToSchemaStoreOrg>
```

Comment:
>PreviousSchemaVersionAsPublishedToSchemaStoreOrg [](start = 5, length = 48)

btw - we need to go back and unlist older packages, since we don't have an automated task to unlist older versions (we should open an issue to resolve this)

Reply:
[https://github.com/microsoft/sarif-sdk/issues/1496](https://github.com/microsoft/sarif-sdk/issues/1496)

---
In reply to: [285750365](https://github.com/microsoft/sarif-sdk/pull/1494#discussion_r285750365) [](ancestors = 285750365)